src/js/media-theme-element.js

```diff
@@ -7,7 +7,9 @@ import { camelCase } from './utils/utils.js';
 export * from './utils/template-parts.js';
 
 const observedMediaAttributes = {
+  'media-target-live-window': 'targetLiveWindow',
   'media-stream-type': 'streamType',
+  'audio': 'audio',
```

Comment:
This attr might deserve a rename/rethink. With media container it sets the _layout_ to audio, no matter what the media. i.e. you may want the audio layout even though you're playing a video.

Reply:
good call, in Mux player this is ok but yes can see we'd want the source type in some situations